flink/v1.20/flink-runtime/LICENSE

```diff
@@ -464,47 +425,92 @@ License text:
 
 --------------------------------------------------------------------------------
 
-This binary artifact includes Project Nessie with the following in its NOTICE
-file:
+This binary artifact contains Project Nessie.
 
-| Dremio
-| Copyright 2015-2017 Dremio Corporation
-|
-| This product includes software developed at
-| The Apache Software Foundation (http://www.apache.org/).
+Copyright: Copyright 2015-2017 Dremio Corporation
```

Comment:
Is this copyright correct?

Reply:
It's what I see here: https://github.com/projectnessie/nessie/blob/nessie-0.102.2/NOTICE#L2

Reply:
As we updated Nessie version, I've updated the copyright with https://github.com/projectnessie/nessie/blob/nessie-0.102.4/NOTICE#L5